### ISSUE TITLE
Omnipose Training Throwing Singular Matrix Error

### ISSUE THREAD

**Issue by nikamarand** [kevinjohncutler/omnipose#111]
I have been using Omnipose for over a year now, and have recently needed to train segmentation for certain datasets. I have masks that I have fixed in Napari (using as ground truths) and saved as labeled masks. I have my files organized per your read the docs. I have one monochannel raw tif image file and one masks tiff file all cropped into 512 by 512 px. Both files are 16bit. 

This is the CLI: 
python -m omnipose --train --dir <dir>  --pretrained_model bact_phase_omni --mask_filter _masks --img_filter _img --diameter 0 --learning_rate 0.1 --batch_size 1 --n_epochs 4000 --verbose

The error seems to be with how the images/masks are being transformed, but I am not sure how to fix it or if what I am supplying will not work with the training function. It's throwing a singular matrices error (error shown below in attachement). 

Any advice in how to fix this would be great, thanks! 

<img width="870" alt="Image" src="https://github.com/user-attachments/assets/bb545e2e-ee3f-4aef-81ab-7819edcb63c7" />
